Makefile

```diff
@@ -31,7 +31,7 @@ YQ ?= $(LOCALBIN)/yq
 ## Tool Versions
 KUSTOMIZE_VERSION ?= v3.8.7
 CONTROLLER_TOOLS_VERSION ?= v0.8.0
-KCP_VERSION ?= 0.7.5
+KCP_VERSION ?= 0.8.2
```

Comment:
Yes I can do that next week. I would need to bump controller-runtime dependencies as well and check that things are still working. Another point related is whether we want to rebase controller-runtime fork on a newer upstream version. I don't think it has been done since the fork was created ~5-6 months ago.
I would prefer doing it in separate PRs. Do you see it otherwise?

Reply:
The latest version of controller-runtime (0.13.0) has k8s 1.25 deps (https://github.com/kubernetes-sigs/controller-runtime/blob/v0.13.0/go.mod). But we do have 0.12.3 (https://github.com/kubernetes-sigs/controller-runtime/blob/v0.12.3/go.mod), which support k8s 1.24 and has a few bug fixes that could be useful.

Reply:
I am looking at that again. Actually kcp fork of controller-runtime is already on  0.12.3. I am testing with v0.9.1. If there is no issue I will push the change here.

Reply:
@ncdc tests were good I have pushed the change with v0.9.1